SECP-service/src/main/java/com/visucius/secp/models/Group.java

```diff
@@ -7,6 +7,14 @@
 
 @Entity
 @Table(name = "Groups")
+@NamedQueries(
+    {
+        @NamedQuery(
+            name = "com.visucius.secp.models.Group.findByName",
+            query = "from Group g where g.name = :name"
+        )
+    }
+)
 public class Group {
 
```

Comment:
I thought we agreed that a group should have relations to a permission table. I cannot see that here.